linalg/solvers.hpp

```diff
@@ -169,10 +169,11 @@ class IterativeSolver : public Solver
 
    ///@}
 
-   /// Return the dot product of @a x and @a y
-   double Dot(const Vector &x, const Vector &y) const;
+   /// Return the standard inner (dot) product of @a x and @a y
```

Comment:
Maybe change `standard` to L2, not sure what is more understandable 🤔

Reply:
I went with `standard (L2)` - let me know what you think.

Reply:
Yeah, that solves the dilemma 😄